pytorch-job/pytorch-operator/overlays/application/application.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: pytorch-operator-application
+spec:
+  type: pytorch-operator
+  componentKinds:
+  - group: core
+    kind: ConfigMap
+  - group: apps
+    kind: Deployment
+  - group: core
+    kind: ServiceAccount
```

Comment:
just wonder why we ignore other k8s resource here, such as clusterrole, clusterrolebinding, service and so on?

Reply:
I think it is because those resource types are not namespaced.

Reply:
yes correct
We would need to expand the application-controller to include cluster resources